### PR TITLE
[TRA-1694] Improve performance on Gitignore

### PR DESCRIPTION
**Pull request by filipechagas**
[TRA-1694](https://toptal-core.atlassian.net/browse/TRA-1694)

## Description

Move fonts from css to header; Asynchronously load remote scripts and move them to the bottom of the body tag; Set image sizes to reduce layout shifts; Remove unused fonts from css.

## How to test

- docker-compose -f ./docker-compose-dev.yml build
- docker-compose -f ./docker-compose-dev.yml up
- go to [http://localhost:8080](http://localhost:8080)
- verify that the app is still overall working
- verify e2e tests are passing with `yarn test`
- use browser performance tools with webvitals enabled to check for issues

### CODE REVIEW

**Review by MarianoSalerno — APPROVED** [review]
Works locally and tests passed :+1:

File: Resources/Views/index.leaf [hunk 1]
```diff
@@ -28,38 +34,22 @@
 
     <link rel="canonical" href="#(canonicalUrlString)" />
 
-    <link rel="stylesheet" type="text/css" href="external/select2-4.0.13/select2.min.css">
+    <link href='css/fonts/ProximaNova-Regular.woff2' rel='stylesheet' type='text/css'>
+    <link href='css/fonts/ProximaNova-Semibold.woff2' rel='stylesheet' type='text/css'>
```
Comment (gobr) [inline Resources/Views/index.leaf:38]: You also need to add the other font versions, besides `woff2` there's also `woff` and `opentype`, as it is right now font doesn't load on Firefox.

**Review by gobr — CHANGES_REQUESTED** [review]

File: Resources/Views/index.leaf (hunk 1 above)
Comment (filipechagas) [reply Resources/Views/index.leaf:36]: You're right! Forgot to add the other two types.
And you're also right about the preload.
Thank you, Diego! Spot on as always!

**Review by gobr — APPROVED** [review]
👍 LGTM! Nice work!

**Pull request merged**